MdeModulePkg/Core/Dxe/Gcd/Gcd.c

```diff
@@ -690,6 +691,8 @@ ConverToCpuArchAttributes (
     CpuArchAttributes |= EFI_MEMORY_UCE;
   } else if ((Attributes & EFI_MEMORY_WP) == EFI_MEMORY_WP) {
     CpuArchAttributes |= EFI_MEMORY_WP;
+  } else if ((Attributes & EFI_MEMORY_SP) == EFI_MEMORY_SP) {

```

Comment:
This is not needed, right, because EFI_MEMORY_SP is already part of EFI_MEMORY_ATTRIBUTE_MASK

Reply:
Removed